cloud/endagaweb/templates/dashboard/password_change.html

```diff
@@ -0,0 +1,38 @@
+{% extends "dashboard/layout.html" %}
+{% comment %}
+Copyright (c) 2016-present, Facebook, Inc.
+All rights reserved.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree. An additional grant
+of patent rights can be found in the PATENTS file in the same directory.
+{% endcomment %}
+{% load apptags %}
+{% load crispy_forms_tags %}
+
+{% block title %} {% tmpl_const "SITENAME" %} | Monitor Usage, Pay Bills {% endblock %}
+
+{% block content %}
+
+<div class="row">
+  <div class="col-xs-12 page-header">
+    <h3>Hi {{ user_profile.display_name }}!</h3>
+  </div>
+</div>
+
+<div class="row">
+  
+  <div class='col-xs-12 col-sm-6 col-md-4 col-xl-2'>
+    <h4>Change Password</h4>
+    {% crispy change_pass_form %}
+
+    {% for message in messages %}
```

Comment:
Another 'for' loop that I don't understand. If this really is a loop there should be a comment.